File: WebApp/ClientApp/src/domain/groups/models/groupBlank.ts
import { Group } from "./group"

export interface GroupBlank{
    id: string | null,
    name: string | null
}

export namespace GroupBlank{
    export function getDefault(): GroupBlank{
        return {
            id: null,
            name: null
        }
    }

    export function fromGroup(group: Group): GroupBlank{
        return {
            id: group.id,
            name: group.name
        } 
    }
}

File: Domain/Groups/GroupBlank.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Groups
{
    public class GroupBlank
    {
        public Guid? Id { get; set; }
        public String? Name { get; set; }
        public Boolean? IsRemoved { get; set; }
    }
}
